<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>third_form</class>
 <widget class="QDialog" name="third_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>How To Play</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>470</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic Md BT</family>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>third_form</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>third_form</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
